apm-sniffer/apm-agent-core/src/main/resources/META-INF/services/org.apache.skywalking.apm.agent.core.boot.BootService

```diff
@@ -32,4 +32,5 @@ org.apache.skywalking.apm.agent.core.profile.ProfileTaskExecutionService
 org.apache.skywalking.apm.agent.core.meter.MeterService
 org.apache.skywalking.apm.agent.core.meter.MeterSender
 org.apache.skywalking.apm.agent.core.context.status.StatusCheckService
-org.apache.skywalking.apm.agent.core.remote.LogReportServiceClient
\ No newline at end of file
+org.apache.skywalking.apm.agent.core.remote.LogReportServiceClient
+org.apache.skywalking.apm.agent.core.conf.dynamic.ConfigurationDiscoveryService
```

Comment:
Adding service should affect the UT, some tests should fail.

Reply:
fixed.